spec/lib/heartcheck/services/response_formatter_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe Heartcheck::Services::ResponseFormatter do
+  describe "#execute" do
+    let(:check_01) { { dummy1: { status: :ok } , time: 1100 } }
+    let(:check_02) { { dummy2: { status: :ok } , time: 100 } }
+    let(:response_formatter_true) { described_class.new(true) }
+    let(:response_formatter_false) { described_class.new(false) }
+    
+    it "returns hash format" do
+      expect(response_formatter_true.execute([check_01,check_02])).to be_eql(
+        {:dummy1=>{:status=>:ok, :time=>1100},
+         :dummy2=>{:status=>:ok, :time=>100}})
+    end
+
+    it "returns array format" do
```

Comment:
I think you could append to the `it` text something like `when response_formatter is true`, otherwise it can be not so clear why is returns a hash or an array.

Reply:
Done!